t/sanity.t

```diff
@@ -1268,3 +1268,106 @@ failed to run healthcheck cycle
 --- error_log
 bad thing!
 --- timeout: 4
+
+
+
+=== TEST 14: health check with ipv6 backend (good case), status ignored by default
+--- http_config eval
+"$::HttpConfig"
+. q{
+upstream foo.com {
+    server 127.0.0.1:12354;
+    server [::1]:12355;
+    server [0:0::1]:12356 backup;
+}
+
+server {
+    listen 12354;
+    location = /status {
+        return 200;
+    }
+}
+
+server {
+    listen [::1]:12355;
+    location = /status {
+        return 404;
+    }
+}
+
+server {
+    listen [0:0::1]:12356;
+    location = /status {
+        return 503;
+    }
+}
+
+lua_shared_dict healthcheck 1m;
+init_worker_by_lua '
```

Comment:
For newly added tests, we should use the more readable `init_worker_by_lua_block {}` form instead :) Will you make the change? Thanks a lot!